.golangci.yml

```diff
@@ -0,0 +1,62 @@
+run:
+  timeout: 5m
+  modules-download-mode: readonly
+
+linters-settings:
+  goconst:
+    min-len: 2
```

Comment:
It might be worth to increase that to 3 and don't `const` strings like `, `. What do you think?

Reply:
I didn't feel like it was my place to suggest it, but I totally agree.